prettier.config.mjs

```diff
@@ -0,0 +1,8 @@
+/** @type {import('prettier').Config} */
+export default {
+  endOfLine: "lf",
```

Comment:
This is ok. This comment is just to highlight a existing problem of defining win or linux env. For win for example the endingLine would be `cr`. We will keep `lf` as we do use wsl to develop ExpressoTS